### PR TITLE
Delete builtin price estimators

### PR DESCRIPTION
**Pull request by MartinquaXD**
# Description
The move to completely colocated quoters will happen in this week's release. If this does not cause any problems all the code for the built-in price estimators can get deleted.

### CODE REVIEW

**Review by Mateo-mro — APPROVED** [review]

File: crates/shared/src/arguments.rs [hunk 1]
```diff
@@ -217,21 +213,6 @@ pub struct Arguments {
...
-    pub paraswap_partner: Option<String>,
-
-    /// The list of disabled ParaSwap DEXs. By default, the `ParaSwapPool4`
-    /// DEX (representing a private market maker) is disabled as it increases
-    /// price by 1% if built transactions don't actually get executed.
-    #[clap(long, env, default_value = "ParaSwapPool4", use_value_delimiter = true)]
-    pub disabled_paraswap_dexs: Vec<String>,
-
     #[clap(long, env)]
     pub zeroex_url: Option<String>,
```
Comment (sunce86) [inline crates/shared/src/arguments.rs:217]: Should this one be removed as well?

**Review by sunce86 — APPROVED** [review]

File: crates/shared/src/arguments.rs (hunk 1 above)
Comment (fleupold) [reply crates/shared/src/arguments.rs:217]: Looks like there are a lot more configuration arguments (related to 1inch, balancer etc. which shouldn't be needed anymore?)

**Review by fleupold — APPROVED** [review]

File: crates/shared/src/arguments.rs (hunk 1 above)
Comment (MartinquaXD) [reply crates/shared/src/arguments.rs]: Right, forgot a few arguments. The balancer arguments are still used by the http price estimator (i.e. legacy solver thingy). Those can be removed when we migrate the last price estimators to the colocated setup.

**Review by squadgazzz — APPROVED** [review]

**Pull request merged**